### PR TITLE
increase ergo for group form

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] increase ergo fo group form

https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1522
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/91
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/181

### CODE REVIEW

File: Backoffice/Form/Type/Component/RoleChoiceType.php [hunk 1]
```diff
@@ -13,15 +15,32 @@ class RoleChoiceType extends AbstractType
...
+     * @param FormView      $view
+     * @param FormInterface $form
+     * @param array         $options
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $parameters = array();
+        $choices = $this->roleCollector->getRoles();
+        foreach($this->parameters as $key => $parameter) {
+            $parameters[$parameter['category']][$parameter['label']][] = array_search(strtoupper($key), array_keys($choices));
```
Comment (itkg-ngilain) [inline Backoffice/Form/Type/Component/RoleChoiceType.php]: There is two many $parameter(s) in the class, it makes the code not easy to understand. Can you find more explicit names ? Which kind of parameter(s) these are ?

File: Backoffice/Form/Type/Component/RoleChoiceType.php [hunk 2]
```diff
@@ -13,15 +15,32 @@ class RoleChoiceType extends AbstractType
...
         $this->name = $name;
+        $this->parameters = $parameters;
+    }
+
+    /**
+     * @param FormView      $view
+     * @param FormInterface $form
+     * @param array         $options
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
```
Comment (alavieille) [inline Backoffice/Form/Type/Component/RoleChoiceType.php]: can you test this method  in RoleChoiceTypeTest

File: BackofficeBundle/Resources/config/role_parameter.yml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+parameters:
+    open_orchestra_backoffice.role:
+        role_access_tree_general_node :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_general_node }
```
Comment (alavieille) [inline BackofficeBundle/Resources/config/role_parameter.yml]: you can use this syntax : 

        role_access_tree_general_node :
          category: open_orchestra_backoffice.left_menu.editorial.title
          label: open_orchestra_role.label_access_general_node

i think this syntax is more explicit

File: BackofficeBundle/Resources/public/less/role.less [hunk 4]
```diff
@@ -0,0 +1,27 @@
+.role-table {
+    border-collapse: collapse;
+	display: table;
+}
+.role-row {
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/less/role.less]: role-row is always in an element with class role-table can you use less syntax to add role-row property in role-table same for other selector

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 5]
```diff
@@ -246,3 +246,31 @@
           data-url="{{ path('open_orchestra_backoffice_internal_link_form') }}"
       ></div>
 {% endblock %}
+{% block oo_role_choice_widget %}
```
Comment (alavieille) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: the form is more readable but still a lot of space, can use an inline component ?

File: BackofficeBundle/Resources/config/role_parameter.yml [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_node }
+        role_access_tree_template :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_template }
+        role_access_create_template :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_template }
+        role_access_update_template :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_template }
+        role_access_delete_template :
+            { category: open_orchestra_backoffice.left_menu.editorial.title, label: open_orchestra_role.label_access_template }
+        role_access_tree_template_flex :
```
Comment (alavieille) [inline BackofficeBundle/Resources/config/role_parameter.yml]: remove this role it's not used in version 1.1

**Comment by itkg-nanne** [comment]
change done @itkg-ngilain , @alavieille

File: Backoffice/Form/Type/Component/RoleChoiceType.php [hunk 7]
```diff
@@ -13,15 +15,35 @@ class RoleChoiceType extends AbstractType
...
     protected $name;
+    protected $rolesClassification;
 
     /**
      * @param RoleCollectorInterface $roleCollector
      * @param string                 $name
+     * @param array                  $rolesClassification
      */
-    public function __construct(RoleCollectorInterface $roleCollector, $name)
+    public function __construct(RoleCollectorInterface $roleCollector, $name, $rolesClassification)
```
Comment (alavieille) [inline Backoffice/Form/Type/Component/RoleChoiceType.php]: specify array in method

File: LogBundle/Resources/config/role_parameter.yml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+parameters:
+    open_orchestra_log.role:
+        role_access_log :
+            { category: open_orchestra_backoffice.left_menu.administration.title, label: open_orchestra_role.label_access_log }
```
Comment (alavieille) [inline LogBundle/Resources/config/role_parameter.yml]: update this yaml with same syntax than other

File: UserAdminBundle/Resources/config/role_parameter.yml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+parameters:
+    open_orchestra_user.role:
+        role_access_user :
+            { category: open_orchestra_backoffice.left_menu.administration.title, label: open_orchestra_role.label_access_user }
```
Comment (alavieille) [inline UserAdminBundle/Resources/config/role_parameter.yml]: same

**Comment by alavieille** [comment]
:+1:

**Pull request merged**